packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -160,18 +160,38 @@ export class Listbox extends FASTElement {
     protected focusAndScrollOptionIntoView(): void {
         if (this.contains(document.activeElement) && this.firstSelectedOption) {
             this.firstSelectedOption.focus();
-            this.firstSelectedOption.scrollIntoView({ block: "nearest" });
+            requestAnimationFrame(() => {
+                this.firstSelectedOption.scrollIntoView({ block: "nearest" });
```

Comment:
Options don't appear to be implemented in Safari - do we know if this errors or gracefully fails in that scenario? My preference is definitely to keep this here, just curious if the absence of it in Safari causes any issues.